dwst/scripts/particles.js

```diff
@@ -20,7 +20,32 @@ const specialChars = [
   '\\',
 ];
 
-export class InvalidParticles extends Error { }
+export class InvalidParticles extends Error {
```

Comment:
Since this class now has methods I think it makes even more sense to write unit tests for the class to make sure that everything is working as expected.

Reply:
Look at the Parsee class tests in PR #291 for an example on how to write tests for a class that implements a data type.